geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/GemfireDataCommandsDUnitTest.java

```diff
@@ -589,14 +588,14 @@ public void doTestBug48013() {
   public void testSelectCommand() {
     setupForSelect();
     doTestGetRegionAssociatedMembersForSelect();
-    doTestSelectWithGfshEnvVariables(true);
-    doTestSelectWithGfshEnvVariables(false);
-    doTestSelectProjection();
-    doTestBug48013();
-    doTestSelectProjectionProcessCommand();
-    doTestSelectProjectionWithNestedField();
-    doTestSelectBeansAsResult();
-    doTestSelectBeansWithNestedFieldAsResult();
+    // doTestSelectWithGfshEnvVariables(true);
+    // doTestSelectWithGfshEnvVariables(false);
+    // doTestSelectProjection();
+    // doTestBug48013();
+    // doTestSelectProjectionProcessCommand();
+    // doTestSelectProjectionWithNestedField();
+    // doTestSelectBeansAsResult();
+    // doTestSelectBeansWithNestedFieldAsResult();
```

Comment:
Can these be deleted? If they need to stay, could you add a comment as to why they're commented out.

Reply:
this is an accidental change that needs to be reverted.